src/components/SectioningContent/SectioningContent.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { FC, PropsWithChildren, useContext } from 'react'
+import styled from 'styled-components'
+
+import { LevelContext } from './levelContext'
+
+const SectioningContent: FC<
+  PropsWithChildren<{
+    className?: string
+    as?: 'article' | 'aside' | 'nav' | 'section'
```

Comment:
IMO: 出自が分からなくならないように貼っておきたい

```suggestion
    // via https://html.spec.whatwg.org/multipage/dom.html#sectioning-content
    as?: 'article' | 'aside' | 'nav' | 'section'
```